app/src/components/toggleTip/toggleTip.tsx

```diff
@@ -0,0 +1,142 @@
+import { useState, useRef } from "react";
+import { Text, Box, Icon } from "@nypl/design-system-react-components";
+import { Button } from "@chakra-ui/react";
+
+export const ToggleTip = ({
+  text,
+  toggleTipContent,
+}: {
+  text: string;
+  toggleTipContent: string;
+}) => {
+  const [isClickVisible, setIsClickVisible] = useState(false);
+  const [isHoverFocusVisible, setIsHoverFocusVisible] = useState(false);
+  const liveRegionRef = useRef<HTMLSpanElement | null>(null);
+
+  const addLiveRegion = () => {
+    if (liveRegionRef.current) {
+      setTimeout(() => {
+        liveRegionRef.current!.textContent = toggleTipContent;
+      }, 100);
+    }
+  };
+
+  const removeLiveRegion = () => {
+    if (liveRegionRef.current) {
+      setTimeout(() => {
+        liveRegionRef.current!.textContent = "";
+      }, 100);
+    }
+  };
+
+  return (
+    <Box as="span" display="inline-flex" alignItems="center">
+      {text}
+      <Box position="relative" display="inline-block">
+        <Button
+          aria-label="More info"
+          lineHeight="1"
+          background="unset"
+          padding="0 !important"
+          marginBottom="xxs"
+          marginLeft="xxs"
+          height="unset"
+          minWidth="unset"
+          _hover={{
+            background: "unset",
+            outline: "2px solid var(--nypl-colors-ui-link-primary)",
+          }}
+          onClick={() => {
+            setIsClickVisible((prev) => !prev);
+            setIsHoverFocusVisible(false);
+            if (!isClickVisible) {
```

Comment:
There's a risk of not using the right value for `isClickVisible` since it's updated two lines before and it might not be updated once this line is called. I don't see any issues while testing though but just pointing this out.